ApplyToBecomeInternal/DocumentGeneration/DocumentGeneration.csproj

```diff
@@ -2,8 +2,13 @@
 
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
+        <CodeAnalysisRuleSet>..\.sonarlint\dfe-digital_a2b-internalcsharp.ruleset</CodeAnalysisRuleSet>
     </PropertyGroup>
 
+    <ItemGroup>
+      <AdditionalFiles Include="..\.sonarlint\dfe-digital_a2b-internal\CSharp\SonarLint.xml" Link="SonarLint.xml" />
```

Comment:
Why is SonarLint being added to to DocGen?

Reply:
This was (in retrospect) an auto-update of the VS plugin.

Reply:
![image](https://user-images.githubusercontent.com/89836563/207913579-f4f57efb-d85c-44f9-89a8-9b3d3882ed26.png)
It was this ☝️ hadn't realised that it affected the project :(